src/callbacks.jl

```diff
@@ -585,14 +585,15 @@ end
 # rough implementation, needs multiple type handling
 # always ensures that if r = bisection(f, (x0, x1))
 # then either f(nextfloat(r)) == 0 or f(nextfloat(r)) * f(r) < 0
-function bisection(f, tup, tdir; maxiters=1000)
+function bisection(f, tup, tdir ; maxiters=1000)
```

Comment:
Just had one additional idea: Could we just make this
```suggestion
function bisection(f, tup, forward; maxiters=1000)
```
instead? It seems `tdir` is never actually used apart from the evaluation `isone(tdir)`.